include/exadg/grid/grid.h

```diff
@@ -81,6 +81,11 @@ class Grid
   std::shared_ptr<dealii::Mapping<dim> const>
   get_mapping() const;
 
+  /**
+   * dealii::Triangulation::MeshSmoothing
+   */
+  typename dealii::Triangulation<dim>::MeshSmoothing mesh_smoothing;
```

Comment:
Do we need to store this info?

Reply:
It is not only used in the constructor, but also in do_create_triangulation()

Reply:
I have only created the parameter to avoid code duplication, since `mesh_smoothing` is needed in the constructor of the grid class and in `do_create_triangulation`. I can change it and paste a copy of the if-else in `do_create_triangulation`.

Reply:
copying the code is not an option. The only option I see is a small private function get_mesh_smoothing() returning mesh_smoothing.

Reply:
Then, the member variable can be removed as suggested by @peterrum